### PR TITLE
Migration guide updates

### PR DESCRIPTION
**Pull request by bhall-ctre**
- Add section for control types (fixes https://github.com/CrossTheRoadElec/PhoenixPro-Documentation/issues/48)
- Add section for status frame update frequency
- Add section for feature replacements
- Add section for MPE (currently not supported by Pro)

### CODE REVIEW

File: source/docs/api-reference/api-usage/migration-guide/configuration-guide.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+         m_motor.configGetSupplyCurrentLimit(supplyCurLim);
+
+   .. tab-item:: C++
+      :sync: C++
+
+      .. code-block:: cpp
+
+         // a limited number of configs have ConfigGet* methods;
+         // for example, you can get the supply current limits
+         SupplyCurrentLimitConfiguration supplyCurLim{};
```
Comment (CoryNessCTR) [inline source/docs/api-reference/api-usage/migration-guide/configuration-guide.rst:140]: Add configs:: to this snippet

We also don't have a supply current limit config group anymore, everything is under `CurrentLimitsConfigs`

File: source/docs/api-reference/api-usage/migration-guide/status-signals-guide.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+         // units are rotations, uses the units library
+         auto rotorPos = rotorPosSignal.GetValue();
+
+         // get latency of the signal
+         auto rotorPosLatency = rotorPosSignal.GetTimestamp().GetLatency();
+
+         // synchronously wait 20 ms for new data
+         rotorPosSignal.WaitForUpdate(20_ms);
+
+Changing Update Frequency
```
Comment (CoryNessCTR) [inline source/docs/api-reference/api-usage/migration-guide/status-signals-guide.rst:85]: Add in "(Status Frame Period)" to reference v5 documentation header

File: source/docs/api-reference/api-usage/migration-guide/control-requests-guide.rst [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+     - PositionDutyCycle
+
+   * - Velocity
+     - VelocityDutyCycle
+
+   * - MotionMagic
+     - MotionMagicDutyCycle
+
+   * - Closed-loop + Voltage Compensation
+     - {ClosedLoop}Voltage
```
Comment (CoryNessCTR) [inline source/docs/api-reference/api-usage/migration-guide/control-requests-guide.rst:204]: I think it's more correct to encompass names to be replaced with arrow-brackets, so it should be `<ClosedLoop>Voltage`

Typically it's done this way for things like linux help docs.

**Review by CoryNessCTR — CHANGES_REQUESTED** [review]

File: source/docs/api-reference/api-usage/migration-guide/configuration-guide.rst (hunk 1 above)
Comment (bhall-ctre) [reply source/docs/api-reference/api-usage/migration-guide/configuration-guide.rst:140]: This is the Phoenix 5 code block

**Comment by CoryNessCTR** [comment]
Possible method to columnize the Pro/v5 sections to clearly delineate what is pro and what is v5
https://stackoverflow.com/questions/56749718/how-to-make-2-columns-with-sphinx

File: source/docs/api-reference/api-usage/migration-guide/control-requests-guide.rst (hunk 3 above)
Comment (CoryNessCTR) [reply source/docs/api-reference/api-usage/migration-guide/control-requests-guide.rst:204]: For now we'll leave it as curly braces, and potentially move them to using arrow brackets based on feedback

**Review by CoryNessCTR — APPROVED** [review]

**Pull request merged**